### PR TITLE
Add psalm

### LINKED ISSUE #49: Setup psalm

### PR DESCRIPTION
**Pull request by come-nc**
Fix #49

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,7 +16,9 @@
 	"require-dev": {
 		"phpunit/phpunit": "^9",
 		"nextcloud/coding-standard": "^1.0.0",
-		"nextcloud/ocp": "dev-master"
+		"nextcloud/ocp": "dev-master",
+		"vimeo/psalm": "^5.18",
+		"sabre/dav": "4.5.0"
```
Comment (nickvergessen) [inline composer.json:21]: you use it directly?

File: psalm.xml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<psalm
+	errorLevel="4"
```
Comment (nickvergessen) [inline psalm.xml:3]: add `phpVersion="8.0"` ?

File: composer.json (hunk 1 above)
Comment (come-nc) [reply composer.json:21]: Yes: https://github.com/nextcloud/ldap_contacts_backend/blob/main/lib/Model/Card.php#L28-L32

File: lib/AppInfo/Application.php [hunk 3]
```diff
@@ -34,25 +34,30 @@
 
 require_once __DIR__ . '/../../vendor/autoload.php';
 
-class Application extends App implements IBootstrap {
+class Application extends App implements IBootstrap
+{
```
Comment (nickvergessen) [inline lib/AppInfo/Application.php:38]: needs a bump of the coding style?

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**